modules/comment-likes.php

```diff
@@ -0,0 +1,185 @@
+			}
+
+			if ( in_array( 'publicize', $active ) && ! in_array( 'sharedaddy', $active ) ) {
+				// we have a sharing page but not the global options area
+				add_action( 'pre_admin_screen_sharing', array( $this->settings, 'sharing_block' ), 20 );
+				add_action( 'pre_admin_screen_sharing', array( $this->settings, 'updated_message' ), -10 );
+			}
+
+			if( ! in_array( 'sharedaddy', $active ) ) {
+				add_action( 'admin_init', array( $this->settings, 'process_update_requests_if_sharedaddy_not_loaded' ) );
+				add_action( 'sharing_global_options', array( $this->settings, 'admin_settings_showbuttonon_init' ), 19 );
+				add_action( 'sharing_admin_update', array( $this->settings, 'admin_settings_showbuttonon_callback' ), 19 );
+				add_action( 'admin_init', array( $this->settings, 'add_meta_box' ) );
+			} else {
+				add_filter( 'sharing_meta_box_title', array( $this->settings, 'add_likes_to_sharing_meta_box_title' ) );
+				add_action( 'start_sharing_meta_box_content', array( $this->settings, 'meta_box_content' ) );
+			}
+
+			add_action( 'save_post', array( $this->settings, 'meta_box_save' ) );
+			add_action( 'edit_attachment', array( $this->settings, 'meta_box_save' ) );
+			add_action( 'sharing_global_options', array( $this->settings, 'admin_settings_init' ), 20 );
+			add_action( 'sharing_admin_update',   array( $this->settings, 'admin_settings_callback' ), 20 );
+		}
+	}
+
+	public function admin_init() {
+		add_filter( 'manage_edit-comments_columns', array( $this, 'add_like_count_column' ) );
+		add_action( 'manage_comments_custom_column', array( $this, 'comment_likes_edit_column' ), 10, 2 );
+		add_action( 'admin_print_styles-edit-comments.php', array( $this, 'enqueue_admin_styles_scripts' ) );
+	}
+
+	public function comment_likes_edit_column( $column_name, $comment_id ) {
+		if ( 'comment_likes' !== $column_name ) {
+			return;
+		}
+
+		$blog_id = Jetpack_Options::get_option( 'id' );
+
+		$permalink = get_permalink( get_the_ID() );
+		?>
+		<a title=""
+		   data-comment-id="<?php echo (int) $comment_id; ?>"
+		   data-blog-id="<?php echo (int) $blog_id; ?>"
+		   class="comment-like-count"
+		   id="comment-like-count-<?php echo (int) $comment_id; ?>"
+		   href="<?php echo esc_url( $permalink ); ?>#comment-<?php echo (int) $comment_id; ?>"
+		>
+			<span class="like-count">0</span>
+		</a>
+		<?php
+	}
+
+	function enqueue_admin_styles_scripts() {
+		wp_enqueue_style( 'comment-like-count', plugins_url( 'comment-likes/admin-style.css', __FILE__ ), array(), JETPACK__VERSION );
+		wp_enqueue_script( 'comment-like-count', plugins_url( 'comment-likes/comment-like-count.js', __FILE__ ), array( 'jquery' ), JETPACK__VERSION );
+	}
+
+	public function add_like_count_column( $columns ) {
+		$columns['comment_likes'] = '<span class="vers"><img title="' . esc_attr__( 'Comment likes', 'jetpack' ) . '" alt="' . esc_attr__( 'Comment likes', 'jetpack' ) . '" src="//s0.wordpress.com/i/like-grey-icon.png" /></span>';
```

Comment:
Also may be worth adding screen reader text instead of relying on alt / title text.